kinder/pkg/build/alter/alter.go

```diff
@@ -27,13 +27,12 @@ import (
 	"github.com/google/uuid"
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
-	"k8s.io/kubeadm/kinder/pkg/exec"
-
```

Comment:
present in multiple files in the diff; did the editor auto-remove this line? 
normally we keep the external / k8s imports separate.

Reply:
fixed on all files.